include/ck/ck.hpp

```diff
@@ -123,7 +123,7 @@
 #define CK_EXPERIMENTAL_USE_BUFFER_LOAD_OOB_CHECK_OFFSET_TRICK 0
 #endif
 #define CK_EXPERIMENTAL_USE_BUFFER_STORE_OOB_CHECK_OFFSET_TRICK 1
-#define CK_EXPERIMENTAL_USE_BUFFER_ATOMIC_ADD_OOB_CHECK_OFFSET_TRICK 1
+#define CK_EXPERIMENTAL_USE_BUFFER_ATOMIC_ADD_OOB_CHECK_OFFSET_TRICK 0
```

Comment:
This will influence all the kernels. Need to know why have to change it

Also need a compiler ticket to track the issue

Reply:
I'll work on the compiler ticket

Reply:
https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/616